### PR TITLE
Fixes the cluster scripts after python migration

### PR DESCRIPTION
**Pull request by pascal-roth**
# Description

Fixes cluster setting after merge of [python conversion](https://github.com/isaac-sim/IsaacLab/commit/f565c33d7716db1be813b30ddbcf9321712fc497)

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have updated the changelog and the corresponding version in the extension's `config/extension.toml` file
- [x] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

### CODE REVIEW

**Comment by Mayankm96** [comment]
Small todo: Can you fix the list here with an enter: https://isaac-sim.github.io/IsaacLab/source/deployment/cluster.html#configuring-the-cluster-parameters

**Comment by pascal-roth** [comment]
@Mayankm96 should be ready

File: docker/cluster/cluster_interface.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```
Comment (Mayankm96) [inline docker/cluster/cluster_interface.sh]: Why not /usr/bin/env bash?

It makes sure this works with zsh too then?

**Review by Mayankm96 — APPROVED** [review]

**Comment by Mayankm96** [comment]
@hhansen-bdai merging this MR without your approval as it is blocking the cluster usage on our side. The changes are minimal.

**Pull request merged**